public/js/helpers/chartsHelpers.js

```diff
@@ -9,7 +8,6 @@ const createPartialsList = (series) => {
 };
 
 // Sums the corresponding y values in the datasets of a series, to get the stacked y totals
-
 const yTotals = (partialsList) => {
```

Comment:
Minor point, but this could be a slightly more descriptive function name so usage is clearer